packages/rich-text/src/plugins/Hyperlink/createHyperlinkPlugin.ts

```diff
@@ -2,17 +2,13 @@ import * as React from 'react';
 
 import { FieldExtensionSDK, Link } from '@contentful/app-sdk';
 import { INLINES } from '@contentful/rich-text-types';
-import { AnyObject, KeyboardHandler, HotkeyPlugin } from '@udecode/plate-core';
+import { AnyObject, HotkeyPlugin } from '@udecode/plate-core';
 import isHotkey from 'is-hotkey';
 
 import { isLinkActive, unwrapLink } from '../../helpers/editor';
 import { transformRemove } from '../../helpers/transformers';
-import {
-  RichTextPlugin,
-  CustomRenderElementProps,
-  CustomElement,
-  RichTextEditor,
-} from '../../types';
+import { PlatePlugin, KeyboardHandler } from '../../internal/types';
+import { CustomRenderElementProps, CustomElement } from '../../internal/types';
```

Comment:
We should not have `CustomRenderElementProps` or `CustomElement` as explained above.